### PR TITLE
[13.0][MIG] crm_phonecall_planner: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
**Depends on** https://github.com/OCA/partner-contact/pull/852

cc @tecnativa TT21624

### CODE REVIEW

**Review by manuelcalerosolis — APPROVED** [review]
Functional and code review!

**Comment by pedrobaeza** [comment]
Please fix Travis and runbot

File: crm_phonecall_planner/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (skeller1) [inline crm_phonecall_planner/__init__.py]: Please change all license links to https

File: crm_phonecall_planner/readme/ROADMAP.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+* Incompatible with ``crm_phonecall_summary_predefined`` addon.
```
Comment (skeller1) [inline crm_phonecall_planner/readme/ROADMAP.rst:1]: Nice to have: Maybe you could solve this open roadmap point before merging. With PR #327 extending crm_phonecall is easier. Maybe you have an idea to eliminate this problem.

**Review by skeller1 — CHANGES_REQUESTED** [review]

File: crm_phonecall_planner/readme/ROADMAP.rst (hunk 2 above)
Comment (Tardo) [reply crm_phonecall_planner/readme/ROADMAP.rst:1]: Sorry, i don't see how to use these improvements to remove this incompatibility.

**Review by skeller1 — APPROVED** [review]

**Pull request merged**